submissions/dmytro-botnarenko/html-css-popup/index.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <title>Hooli-style Popup</title>
+  </head>
+  <body>
+    <div class="wrapper">
+      <header class="header">
+        <nav>
+          <ul class="navigation__menu">
+            <li class="navigation__list">
```

Comment:
li - list item. Class `navigation__list` seems not correct naming

Reply:
Made `navigation__item`. Now it's `navigation__menu` -> `navigation__item` -> `navigation__link` chain. Is it ok?

Reply:
Nice, I like it:) If you are using BEM it should have nesting structure like nav-menu nav-menu_item nav-menu_item_link